optimum/exporters/openvino/convert.py

```diff
@@ -329,19 +322,8 @@ def export_pytorch(
     logger.info(f"Using framework PyTorch: {torch.__version__}")
     output = Path(output)
 
-    if stateful:
-        if is_transformers_version("<", "4.36") or is_torch_version("<", "2.1.1"):
-            COLOR_RED = "\033[1;31m"
-            COLOR_RESET = "\033[0m"
-            logger.warning(
-                COLOR_RED
-                + "[WARNING] For good performance with stateful models, transformers>=4.36.2 and PyTorch>=2.1.1 are required. "
-                f"This Python environment has Transformers {_transformers_version} and PyTorch {_torch_version}. "
-                "Consider upgrading PyTorch and Transformers, for example by running "
-                "`pip install --upgrade --upgrade-strategy eager optimum[openvino,nncf]`, and export the model again"
-                + COLOR_RESET
-            )
-
+    is_model_stateful = hasattr(model, "use_bettertransformer") and model.use_bettertransformer is True
```

Comment:
not sure that this is correct assumption. there are some models where we do not need to use bettertransformers (e.g. llama support removed from bettertransformers in the latest optimum, because model support sdpa natively and I'm sure that in future number of such models will increase